test/e2e/resourceinterpreter_test.go

```diff
@@ -422,4 +422,208 @@ end`,
 			}
 		})
 	})
+
+	ginkgo.Context("InterpreterOperation Retain testing", func() {
+		var waitTime time.Duration
+		var updatedPaused bool
+
+		ginkgo.BeforeEach(func() {
+			waitTime = 5 * time.Second
+			updatedPaused = true
+			customization = testhelper.NewResourceInterpreterCustomization(
+				"interpreter-customization"+rand.String(RandomStrLength),
+				configv1alpha1.CustomizationTarget{
+					APIVersion: "apps/v1",
+					Kind:       "Deployment",
+				},
+				configv1alpha1.CustomizationRules{
+					Retention: &configv1alpha1.LocalValueRetention{
+						LuaScript: `
+function Retain(desiredObj, observedObj)
+  desiredObj.spec.paused = observedObj.spec.paused
+  return desiredObj
+end`,
+					},
+				})
+		})
+
+		ginkgo.It("Retain testing", func() {
+			ginkgo.By("update workload's spec.paused to true", func() {
+				clusterClient := framework.GetClusterClient(targetCluster)
+				gomega.Expect(clusterClient).ShouldNot(gomega.BeNil())
+				var memberDeploy *appsv1.Deployment
+				framework.WaitDeploymentPresentOnClusterFitWith(targetCluster, deployment.Namespace, deployment.Name,
+					func(deployment *appsv1.Deployment) bool {
+						memberDeploy = deployment
+						return true
+					})
```

Comment:
This means get deploy from member cluster? Using `WaitDeploymentPresentOnClusterFitWith` function confuses me~

Reply:
yes, this mean get deploy from member cluster